### ISSUE TITLE
Why not using the blockchain to approve the certificates?

### ISSUE THREAD

**Issue by piffpaffpuff** [admin-ch/CovidCertificate-Documents#7]
To me, that would have been a perfect fit for such cases as certificates. 

> Verhindert wird dies (Fälschung) jedoch mit einer kryptografischen Signierung durch eine vom Bund autorisierte Person, sei dies eine Ärztin, ein Apotheker oder jemand im Impfzentrum.

Any random medical health person is safer than a public blockchain? From a technological standpoint, that sounds pretty weird.

**Comment by ghost** [comment]
Blockchain ... pls...

**Comment by piffpaffpuff** [comment]
> Blockchain ... pls...

Can you elaborate that please?

**Comment by herpaderpaldent** [comment]
![image](https://user-images.githubusercontent.com/6583519/120353732-24d19d00-c302-11eb-9430-cc38e9aee57e.png)
ref: https://xkcd.com/2267/ 

>Can you elaborate that please? 

if you have to ask this question, you have not invested the tiniest amount of thoghts into either the usecase nor the technology. There is not a single business case that yet had proven to usefulness of a blockchain. It's an IT Buzzword, such as cloud.
Tl;dr: No.

**Comment by piffpaffpuff** [comment]
Okay, thanks for the clarification. I think it‘s best to close that issue.
